### DISCUSSION TITLE
Your Seq trial is nearly finished

### DISCUSSION THREAD

**Discussion by MHR00** [datalust/seq-tickets#2014] [General]
Hi, 
After the end of trial , are the logs still registered in Seq or not?

**Comment by nblumhardt** [comment]
Hi Mohammad, thanks for reaching out.

At the conclusion of a trial, Seq will continue collecting events, but queries/searches will be unavailable until either a subscription is purchased, or (if you're able to limit access to a single person) the server is downgraded to the Individual tier.

Hope this helps, please get in touch if you need any further info.

Best regards,
Nick

**Comment by MHR00** [comment]
thanks for your answer
